index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
It looks like you have two copies of your index.html, one in the root and one in pages/index.html. They also look to have slightly different content. Look out for this accidentally duplication in the future, it's an easy mistake to make!